README.md

```diff
@@ -43,13 +43,27 @@ These instructions give the most direct path to a working Swift
 development environment. Options for doing things differently are
 discussed below.
 
-
 ### System Requirements
 
 macOS, Ubuntu Linux LTS, and the latest Ubuntu Linux release are the current
 supported host development operating systems.
 
-For macOS, you need [Xcode 9.0 beta 4](https://developer.apple.com/xcode/downloads/).
+To build for macOS, you always need the latest version of Xcode, currently
+[Xcode 9.0 beta 4](https://developer.apple.com/xcode/downloads/). You will
```

Comment:
I think it'd be good to really make clear that the latest supported is beta 4. I have a lot of issues when I started because Swift wasn't compiling on beta 4 (and this doc said beta 3). I assumed it was all my fault 😬